x/subspaces/spec/02-concepts.md

```diff
@@ -23,6 +23,8 @@ An optional description of what the subspace is about.
 ### Treasury
 The treasury address represents the wallet of the subspace itself. This can be used for different reasons such as verifying a subspace with external applications (to prove its authenticity), or paying for fees when executing some smart contracts. 
```

Comment:
It can still be done by sending a `MsgCreateProfile` and then a `MsgLinkApplication` on behalf of the treasury. This is a way to verify the authenticity of a subspace: by linking the profile associated with the treasusy with an official website of the application.

Reply:
In order to link an app to the treasury account, signing message by treasury account  private key is required, or the verification process would be failed.
https://github.com/desmos-labs/desmos/blob/master/x/profiles/keeper/relay_app_links.go#L193

But currently treasury account is auto generated by its subspace ID, there is no way to get the treasury account.

Reply:
Oh you're right :disappointed: I've fixed the description